firewood/src/merkle/node.rs

```diff
@@ -157,6 +157,7 @@ impl NodeType {
 pub struct Node {
     pub(super) root_hash: OnceLock<TrieHash>,
     encoded: OnceLock<Vec<u8>>,
+    is_encoded_longer_than_hash_len: OnceLock<bool>,
```

Comment:
We took this out because the `OnceLock` was really slow. If we need to add it back in, then we should create a structure that has both `Vec<u8>` and `bool` in it, and wrap that in the `OnceLock`. There is no point having two of them.

Reply:
We will do this in a followup, if we find it matters.